edpop_explorer/edpopxshell.py

```diff
@@ -19,6 +19,7 @@
     VDLiedReader,
     KBReader,
     STCNReader,
+    STCNPersonsReader,
```

Comment:
It is confusing indeed, but the catalogues are named "STCN persons" and "STCN printers", respectively (even though in practice, it seems that most of the publishers/printers are private persons rather than organisations...). I clarified this now in the docstring.